.travis.yml

```diff
@@ -12,7 +17,7 @@ before_script:
 script:
 - |
   # get ourselves a MIR-ful libstd
-  xargo/build.sh
+  cd xargo && RUSTFLAGS='-Zalways-encode-mir' xargo build && cd ..
```

Comment:
Wait, macOS has no readlink? Or just doesn't support the flag?
The flag is just needed to make the script work even when called though a symlink. I add a line like this in the beginning of every shell script I write, but we could simplify this one to make it more portable. Just `cd "$(dirname "$0")"` should work.